spec/factories/registration.rb

```diff
@@ -25,5 +25,13 @@
     trait :sole_trader do
       business_type { "soleTrader" }
     end
+
+    # This is the minimum needed to be able to generate a confirmation email
+    # and its attachments sucessfully
+    trait :emailed do
```

Comment:
If this is the minimum "required" to generate an email I find the naming not quite accurate. What do you think about de-coupling this trait's name from the purpose it has been generated for and go for a more general naming? I mean, this trait actually creates a `valid` or `submitted` or `complete` Registration object, which by default will be "emailable" if that makes sense :D

Reply:
Completely get your point, only issue is just adding these attributes doesn't make it any of those i.e.

- it's not `valid` because its missing a bunch of data
- it could be `submitted` but the definition of a submitted registration is that the record exists, so you could argue no data whatsoever is needed to meet this definition
- I think `complete` is more relevant to a transient_registration, as again we're back to the idea that a `submitted` or `complete` transient_registration is simply a registration. Also IMHO in the terms of the factory I'd assume `complete` refers to a fixture with all data populated

So I'm stuck on naming. Perhaps `emailable` is a better definition (and according to the internet it is a word no matter what my spell checker says! https://en.oxforddictionaries.com/definition/emailable) but I'm open to anything better.

Reply:
Naming things is always the difficult part! :D I think `emailable` is the best choice here :)